docs/node-operators/snark-workers/getting-started.mdx

```diff
@@ -1,21 +1,87 @@
 ---
-title: Snark Workers
+title: Getting Started
 hide_title: true
 description: How to run a Snark Worker. Snark Workers are essential to keep the network producing transactions in blocks. 
 keywords:
   - connect to network
   - running a snark worker
 ---
 
+# Getting Started 
+
 The Mina protocol is unique because nodes are not required to maintain the full history of the blockchain like other cryptocurrency protocols. By recursively using cryptographic proofs, the Mina protocol effectively compresses the blockchain to constant size. This compression reduces terabytes of data to a few kilobytes.
 
-However, this isn't data encoding or compression in the traditional sense. Mina nodes _compress_ data in the network by generating cryptographic proofs. Node operators play a crucial role in this process by designating themselves as [SNARK-workers](/glossary#snark-worker) that generate [zk-SNARKs](/glossary#zk-snark) for transactions that have been added to blocks.
+However, this isn't data encoding or compression in the traditional sense. Mina nodes _compress_ data in the network by generating cryptographic proofs. Node operators play a crucial role in this process by designating themselves as [SNARK-workers](/glossary#snark-worker) that generate [zk-SNARKs](/glossary#zk-snark) for transactions that have been added to blocks. 
```

Comment:
```suggestion
However, this isn't data encoding or compression in the traditional sense. Mina nodes _compress_ data in the network by generating cryptographic proofs. Node operators play a crucial role in this process by designating themselves as [SNARK workers](/glossary#snark-worker) that generate [zk-SNARKs](/glossary#zk-snark) for transactions that have been added to blocks. 
```

Reply:
do node operators (people) designate themselves? (IIUC, the answer is no) Suggest rewriting for accuracy and clarity. Do the people designate a node to serve as a SNARK worker?